### PR TITLE
CSS File Updates for ConfigPage

### LINKED ISSUE #5: Extract all CSS in JS to CSS files
Wherever we use CSS in JS (like in the following image)...

image

We need to change to using a traditional CSS approach - i.e., give the DOM element a class name (via the `className` attribute), then target that class using CSS in the component's related CSS file. Move any style declarations from in the component TSX file to its CSS file

### PR DESCRIPTION
**Pull request by VincentLeighton**
# Author Section
Dr Mundo
## Description

CSS File Updates for Config Page per AWilliam's request.
## PR Readiness Checklist

- [x] Verified all changes are related to feature described above (no scope creep)
- [x] Checked target and base branch (should be feature branch merging INTO `main`)
- [ ] Added unit tests to cover modified / additional code -----"No test"
- [x] Added assignee

### CODE REVIEW

**Comment by Roughnick** [comment]
So there are CSS files per component, wouldn't we want to have a more central location for all the CSS? So the design stays more consistent as the site gets bigger

**Comment by Wolven531** [comment]
We don't yet have a central design, but as common CSS is added, we will add it to the `src/index.css`. In React, CSS is either done in the component code (a la CSS-in-JS), or in sibling CSS files (one per component); this change makes it so that we steer away from CSS-in-JS. 

We're choosing to do that because CSS-in-JS has some performance implications (though minor, they add up over time) and it will be easier to maintain the site style using CSS files. As an added benefit, CSS selectors will be used to link the styles to the elements, and those same selectors can make writing unit tests much easier

File: src/ConfigPage/ConfigPage.css [hunk 1]
```diff
@@ -1,3 +1,22 @@
 .config-page .btn-update-token {
 	margin-top: 1rem;
 }
+
+.config-page {
+	align-items: center;
+	align-content: center;
```
Comment (Wolven531) [inline src/ConfigPage/ConfigPage.css:7]: I always try to keep my CSS style rules in alphabetic order, so I am going to rewrite these two lines as the following -

```suggestion
	align-content: center;
	align-items: center;
```

File: src/ConfigPage/ConfigPage.tsx [hunk 2]
```diff
@@ -40,14 +40,14 @@ const ConfigPage: FC<ConfigPageProps> = ({ api, defaultSecret = '', defaultToken
 				container
 				alignItems="center"
 				justify="center"
-				style={{ margin: '0 0 1rem 0' }}
+				className="api-container"
 			>
 				<Grid item>API</Grid>
-				<Grid item style={{ margin: '0 1rem 0 1rem' }}>
+				<Grid item >
```
Comment (Wolven531) [inline src/ConfigPage/ConfigPage.tsx:46]: There's an extra space here

```suggestion
				<Grid item>
```

**Review by Wolven531 — CHANGES_REQUESTED** [review]
Leaving a RFC (Request For Changes), but I'm going to commit the changes and approve this PR, then merge it

File: src/ConfigPage/ConfigPage.css [hunk 3]
```diff
@@ -1,3 +1,22 @@
...
+	align-items: center;
+	margin: 0 0 1rem 0;
+}
+
+.config-page .api-container {
+	margin: 0 0 1rem 0;
+}
+
+.config-page .MuiGrid-item{
```
Comment (Wolven531) [inline src/ConfigPage/ConfigPage.css:15]: Space missing here

```suggestion
.config-page .MuiGrid-item {
```

File: src/ConfigPage/ConfigPage.css [hunk 4]
```diff
@@ -1,3 +1,22 @@
...
+	margin: 0 1rem 0 1rem;
+}
+
+.config-page .center-flex{
```
Comment (Wolven531) [inline src/ConfigPage/ConfigPage.css:19]: Space missing here

```suggestion
.config-page .center-flex {
```

**Review by Wolven531 — CHANGES_REQUESTED** [review]
Minor cleanup

**Review by Wolven531 — APPROVED** [review]
Lgtm, 👍

**Pull request merged**